packages/x-charts/src/ChartsXAxis/ChartsXAxis.tsx

```diff
@@ -96,6 +100,7 @@ const defaultProps = {
   disableLine: false,
   disableTicks: false,
   tickSize: 6,
+  tickLabelMinGap: 4,
```

Comment:
I've reduced this to 4px as it seems enough. 

Examples with 4px:

![image](https://github.com/user-attachments/assets/be2b0c19-e58c-4af9-9313-c7ee2bf69dcb)


![image](https://github.com/user-attachments/assets/4ee1db03-16a6-4ad0-bbcf-e520e0ec4d93)


Examples with 8px:

![image](https://github.com/user-attachments/assets/74a3c0c6-4fa0-4728-996a-2d6d52b4370f)

![image](https://github.com/user-attachments/assets/f1550828-24fa-41fd-bbd7-73e55401d0a4)